src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -88,7 +88,7 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
           }>
           <TouchableOpacity onPress={() => setIsTooltipShown(true)}>
             <View>
-              <MemoIcQuestionMark width={normalize(13)} height={normalize(13)} />
+              <MemoIcQuestionMark width={normalize(16)} height={normalize(16)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The size of the MemoIcQuestionMark icon has been increased from 13 to 16. This will make the icon more visible, but ensure that it doesn't disrupt the layout or aesthetics of the surrounding elements.
```diff
-              <MemoIcQuestionMark width={normalize(13)} height={normalize(13)} />
+              <MemoIcQuestionMark width={normalize(16)} height={normalize(16)} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->